packages/@aws-cdk/aws-iotevents/README.md

```diff
@@ -15,9 +15,193 @@
 ---
 <!--END STABILITY BANNER-->
 
-
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+AWS IoT Events is a fully managed service that makes it easy to detect and respond to events from IoT sensors and applications. Events are patterns of data identifying more complicated circumstances than expected, such as changes in equipment when a belt is stuck or motion detectors using movement signals to activate lights and security cameras. Before IoT Events, you had to build costly, custom applications to collect data, apply decision logic to detect an event, and then trigger another application to react to the event. Using IoT Events, it’s simple to detect events across thousands of IoT sensors sending different telemetry data, such as temperature from a freezer, humidity from respiratory equipment, and belt speed on a motor. You simply select the relevant data sources to ingest, define the logic for each event using simple ‘if-then-else’ statements, and select the alert or custom action to trigger when an event occurs. IoT Events continuously monitors data from multiple IoT sensors and applications, and it integrates with other services, such as AWS IoT Core and AWS IoT Analytics, to enable early detection and unique insights into events. IoT Events automatically triggers alerts and actions in response to events based on the logic you define to resolve issues quickly, reduce maintenance costs, and increase operational efficiency.
+
+### Installation
+
+```ts
+import {
+  DetectorModel,
+  Event,
+  Input,
+  State,
+  TransitionEvent,
+} from "@aws-cdk/aws-iotevents";
+```
+
+### Basic usage
+
+A detector model needs at least one state as entry point;
+A basic detector model which doesn't do anything is shown here:
+
 ```ts
-import iotevents = require('@aws-cdk/aws-iotevents');
+import {
+  DetectorModel,
+  Event,
+  Input,
+  State,
+  TransitionEvent,
+} from "@aws-cdk/aws-iotevents";
+
+const detector_model = new DetectorModel(this, "DetectorModel", {
+  modelName: "MyDetector",
+  description: "Detecting things",
+});
+
+const entry_state = new State("EntryState");
```

Comment:
use TS conventions for variable naming so that the examples are idiomatic instead of snake_case